internal/integration/testcase_test.go

```diff
@@ -110,8 +128,17 @@ func (t testcase) createClusterQueueWithCleanup() *buildkite.ClusterQueue {
 			return
 		}
 
-		_, err := t.Buildkite.ClusterQueues.Delete(cfg.Org, cfg.ClusterUUID, *queue.ID)
-		if err != nil {
+		if err := roko.NewRetrier(
+			roko.WithMaxAttempts(5),
+			roko.WithStrategy(roko.Exponential(time.Second, 5*time.Second)),
```

Comment:
I think this is equivalent?
```suggestion
			roko.WithStrategy(roko.Constant(5*time.Second)),
			roko.WithJitter(),
```